build.ps1

```diff
@@ -898,6 +932,41 @@ function Build-SourceKitLSP($Arch)
     }
 }
 
+function Build-Installer()
+{
+  Build-WiXProject toolchain.wixproj -Properties @{
+    DEVTOOLS_ROOT = "$ToolchainInstallRoot\";
+    TOOLCHAIN_ROOT = "$ToolchainInstallRoot\";
+  }
+
+  msbuild $SourceCache\swift-installer-scripts\platforms\Windows\CustomActions\SwiftInstaller\SwiftInstaller.vcxproj -t:restore
+  Build-WiXProject sdk.wixproj -Properties @{
+    PlatformToolset = "v143";
```

Comment:
This will go away with https://github.com/apple/swift-installer-scripts/pull/176

Reply:
The `PlatformToolset` was for the `SwiftInstaller.vcxproj`?

Reply:
Removed